txgithub/api.py

```diff
@@ -40,11 +40,14 @@ class GithubApi(object):
     # - optional user/pass auth (token is not available with v3)
     # - async API
 
-    def __init__(self, oauth2_token, baseURL=None):
+    def __init__(self, oauth2_token, baseURL=None, _reactor=None):
```

Comment:
1 seems an expedient choice to get to 100% coverage, and things can be incrementally improved from there.